cocotb/share/makefiles/simulators/Makefile.activehdl

```diff
@@ -44,9 +44,11 @@ else
    $(error "A valid value (verilog or vhdl) was not provided for TOPLEVEL_LANG=$(TOPLEVEL_LANG)")
 endif
 
+# Always create the DO script
+FORCE_RUNSIM:
 # Create a DO script (Tcl-like but not fully compatible) based on the list of $(VERILOG_SOURCES)
-$(SIM_BUILD)/runsim.do : $(VERILOG_SOURCES) $(VHDL_SOURCES) $(SIM_BUILD)
-	@echo "alib $(RTL_LIBRARY)" >> $@
+$(SIM_BUILD)/runsim.do : $(VERILOG_SOURCES) $(VHDL_SOURCES) $(SIM_BUILD) FORCE_RUNSIM
```

Comment:
This would be then different than riviera and Questa/Modelsim. 

What is interesting this is effectively the case. I am not some expert here and maybe wrong but it looks dependency on `$(SIM_BUILD)` is causing `runsim.do`to be always re-created effectively (check with Moelsim, `sim_build` timestamp is updated). Not sure if this is intended? Maybe it should be done explicitly than? @eric-wieser

Reply:
Locally with Conda on Windows, `sim_build` is not re-created, and `runsim.do` is not updated. I have to delete it first if I want for instance PLUSARGS to change between runs.

Reply:
In any case maybe `.PHONY: $(SIM_BUILD)/runsim.do` be better? Any negative consequences @eric-wieser ?

gnu.org/software/make/manual/html_node/Phony-Targets.html

Reply:
Force target makes more sense to me, and is supported on versions of `make` that don't support `.PHONY`
https://www.gnu.org/software/make/manual/html_node/Force-Targets.html

Reply:
We just have plenty of ".PHONY" other places.